### ISSUE TITLE
Error: TypeError: Input does not meet YAML 1.2 "Core Schema" specification: noCompatMode

### ISSUE THREAD

**Issue by SVynnytska** [fjogeleit/yaml-update-action#534]
Existing GH actions started to fail today, after latest changes in this  repository, we got an error :
```
Error: TypeError: Input does not meet YAML 1.2 "Core Schema" specification: noCompatMode
Support boolean input list: `true | True | TRUE | false | False | FALSE`
```

**Comment by fjogeleit** [comment]
Hey, I will have a look asap. thanks for reporting

**Comment by fjogeleit** [comment]
I added a default value to the `noCompatMode` input. I hope this resolves your issue.

**Comment by rodriwp** [comment]
Hello, I have the same issue here. 

Adding `noCompatMode: false` fixed the issue for me.

**Comment by fjogeleit** [comment]
I released the fix as version `v0.12.3`. Let my know if you still encounter any issue.
